test/SinglePageBrowser.test.js

```diff
@@ -83,6 +83,78 @@ describe.each(['firefox', 'chrome', 'safari'])('Tests on %s', (title) => {
     });
   });
 
+  describe('Check Syl Bounding Box UI', () => {
```

Comment:
Is there a reason this wasn't grouped in with other edit mode tests?

Reply:
i kept it separate since it's a separate module in neon and because the beforeAll and afterAll calls are different